packages/react/src/components/Address/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface AddressProps {
+  children: React.ReactNode;
+  className?: string;
+}
```

Comment:
TypeScript users will not be able to pass "other" props unless we explicitly state that they're supported. We also get `className` for free (as it's a valid prop for any element), so this should be:

```suggestion
interface AddressProps extends React.HTMLAttributes<HTMLElement> {
  children: React.ReactNode;
}
```

Reply:
I was patterning this off of [DescriptionList](https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/DescriptionList/index.tsx), but I believe you are correct. I suspect DescriptionList would have similar issues for someone using TypeScript.